app/dev_mode/templates/dev-page.html

```diff
@@ -70,6 +70,11 @@
             <option name="GB-WLS" value="GB-WLS">GB-WLS</option>
           </select>
         </p>
+        <p>
+            Sexual Identity Flag:
+            <input type="checkbox" name="sexual_identity" class="qa-sexual-identity" value="true">
+        </p>
+
```

Comment:
For this yes, but the variant flags (if used) will probably be different for every survey.